diff --git a/node_modules/@safe-global/safe-core-sdk-utils/dist/src/eip-3770/config.js b/node_modules/@safe-global/safe-core-sdk-utils/dist/src/eip-3770/config.js
index 7dbddf5..4b5630d 100644
--- a/node_modules/@safe-global/safe-core-sdk-utils/dist/src/eip-3770/config.js
+++ b/node_modules/@safe-global/safe-core-sdk-utils/dist/src/eip-3770/config.js
@@ -117,6 +117,8 @@ exports.networks = [
     { chainId: 421611, shortName: 'arb-rinkeby' },
     { chainId: 421613, shortName: 'arb-goerli' },
     { chainId: 534353, shortName: 'scr-alpha' },
+    { chainId: 810180, shortName: 'zklink-nova' },
+    { chainId: 810182, shortName: 'zklink-nova-goerli' },
     { chainId: 11155111, shortName: 'sep' },
     { chainId: 245022926, shortName: 'neonevm-devnet' },
     { chainId: 1313161554, shortName: 'aurora' },
